processing/format_qaqc_driver.py

```diff
@@ -78,21 +84,35 @@ def __init__(self, lookback_h=None, test=False):
         self.stale_count = 0
         self.blacklist_uuid = []
 
-    def send_email_to_amp(self, msg):
+    def send_email_to_amp(self, msg, token, site_id, upload_id=None):
         sender = self.qaqc_processor_email
-        if isinstance(self.amp_team_email, str):
-            receipient = [self.amp_team_email]
-        subject = 'FormatQAQCDriver Abnormal Report'
-        msg = self.email_amp.build_multipart_text_msg(sender,
-                                                      receipient,
-                                                      subject,
-                                                      msg)
-        self.email_amp.send_mail(sender, receipient, msg)
+        receipient = self.amp_team_email
+        timestamp = datetime.datetime.now().strftime('%Y-%m-%d %H:%M:%S')
```

Comment:
Is there a format that has been defined somewhere and you can just reference that?

Reply:
Do you mean the email format or the timestamp format?

Reply:
I use TimestampUtils.JIRA_TS_FORMAT